src/components/ProductSwatch/ProductSwatch.tsx

```diff
@@ -46,7 +45,7 @@ export default function ProductSwatch(props: ProductSwatchProps) {
           selectedVariation,
         })
       ) : (
-        <div>
+        <div onClick={swatchContainerClickHandler} role='button' tabIndex={0}>
```

Comment:
I don't think we'd want this. This is neither a button nor should be navigatable-to using tabs since it's just a container. All we want to do is to prevent clicking into this space to trigger a redirect/click event so throwing a 
```
        // eslint-disable-next-line jsx-a11y/no-static-element-interactions
```
might be more advisable here. Calling our resident a11y expert @mocca102 for his thoughts.